stubs/passlib/passlib/utils/__init__.pyi

```diff
@@ -46,30 +47,41 @@ class SequenceMixin:
     def __ne__(self, other): ...
 
 consteq = compare_digest
+str_consteq = compare_digest
```

Comment:
Ironically, although passlib does `str_consteq = consteq`, it's not the same as the `compare_digest`. Using a current Python, the following happens:

```
def consteq() ...
str_consteq = consteq
from hmac import compare_digest as consteq  # overriding consteq
```

So I think it's correct to keep `str_consteq()` as its own function instead of an alias.